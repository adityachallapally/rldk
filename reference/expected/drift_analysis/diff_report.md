# Divergence Analysis Report

## 📊 Summary

- **Diverged:** No
- **Total divergence events:** 0
- **Signals analyzed:** No obvious cause identified - may be due to numerical instability


## 📁 Files Generated

- **Diff Report:** `reference/expected/drift_analysis/diff_report.md`
- **Drift Card:** `reference/expected/drift_analysis/drift_card.md`
