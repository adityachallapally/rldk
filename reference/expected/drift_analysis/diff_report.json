{
  "version": "1",
  "diverged": false,
  "first_step": null,
  "tripped_signals": [],
  "notes": [
    "No significant divergence detected",
    "Threshold: 3 consecutive violations",
    "Rolling window size: 50"
  ],
  "suspected_causes": [
    "No obvious cause identified - may be due to numerical instability"
  ]
}